Signal/src/ViewControllers/BlockListUIUtils.h

```diff
@@ -44,6 +44,11 @@ typedef void (^BlockActionCompletionBlock)(BOOL isBlocked);
                           contactsManager:(OWSContactsManager *)contactsManager
                           completionBlock:(nullable BlockActionCompletionBlock)completionBlock;
 
+#pragma mark - UI Utils
+
++ (NSString *)formatDisplayNameForAlertTitle:(NSString *)displayName;
++ (NSString *)formatDisplayNameForAlertMessage:(NSString *)displayName;
```

Comment:
Eventually it'd be nice to move these functions to a new `AlertUtils` (or whatever) file.